.gitignore

```diff
@@ -39,5 +39,4 @@ xcuserdata/
 
 [Dd]ebug/
 [Rr]elease/
-x64/
-x86/
\ No newline at end of file
+x64/
```

Comment:
I think it makes sense to get rid of the x64 here, if we are getting rid of the x86.